components/activities/activities-tab/activities-tab.tsx

```diff
@@ -41,29 +35,19 @@ export function ActivitiesTabManager<
   ActivityType extends ActivityListItemTypes[SourceType],
 >(props: ActivitiesTabManagerProps<ActivityListItemTypes>) {
   useStyleSheet(styleSheet);
+
   const orgUserWithRoles$ = useOrgUserWithRoles$();
   const { pushDialog } = useDialog();
   const activityListItems = props.activityListItems ?? DEFAULT_LIST_ITEMS;
 
-  const activityAlertConnection$ = usePollingActivityAlertConnection$<ActivityType, SourceType>(
-    { interval: 2000, limit: 20 },
-  );
-
-  const activeActivities = useSelector(() =>
-    activityAlertConnection$.data?.get()?.alertConnection.nodes.filter((activity) =>
-      activity?.activity
-    ).filter(isActiveActivity) || []
-  );
-
-  // HACK - this is a hack to only open the active prediction
-  // remove once we've cleaned up the prediction component so we can
-  // render past prediction results
-  const firstPrediction = activeActivities.find((activity) => isPrediction(activity.activity));
+  const { activityAlertConnection$, reexecuteActivityConnectionQuery } =
+    usePollingActivityAlertConnection$<ActivityType, SourceType>(
+      { interval: 2000, limit: 20 },
+    );
 
-  const pastActivities = useSelector(() =>
-    activityAlertConnection$?.data?.get()?.alertConnection.nodes
-      .filter((activity) => activity?.activity && !isActiveActivity(activity))
-  );
+  useEffect(() => {
```

Comment:
what's this for? can you comment too

Reply:
yeah, w/ the polling for whatever reason it wasn't fetching on mount so this is used to ensure the activities get pulled as soon as the component renders

Reply:
can probably remove honestly since the tab gets loaded when the menu gets booted up.